### PR TITLE
Fix package installation issues

### PR DESCRIPTION
**Pull request by GarrickHe**
* Bugfix: SSH connection lost during package installation
* Bugfix: Installation hangs due to service start-up order

Fixes open-switch/opx-nas-qos#21

Signed-off-by: Garrick He <garrick_he@dell.com>

### CODE REVIEW

File: scripts/bin/network_restart.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
-service networking restart
+#service networking restart
```
[the remark this answers is not in the document]
Comment (jeff-yin) [reply scripts/bin/network_restart.sh]: Or just change this line to say `#bring up all interfaces except eth0`

**Comment by GarrickHe** [comment]
Pull-request updated, HEAD is now f229de7c71fd856024a87b00a95afb6d192959f8

File: scripts/bin/network_restart.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (GarrickHe) [reply scripts/bin/network_restart.sh]: No problem.

**Comment by theucke** [comment]
🎉

**Pull request merged**